Part2_Mapplots/ex2_3_contour.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Exercise 2.3 Contour plots (cartopy)\n",
```

Comment:
<p>Why replacing subplots instead of axes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I found <code>plt.subplots</code> to be more flexible in general so I started to using only this in my scripts. Do you prefer that I revert some/ most/ all of these changes?</p><p> </p><p>Alternatively I can mention the <code>plt.axes()</code> syntax in 2.0 and say that we will mostly use the other throughout the  course but the users are free to use whatever they prefer?</p>

Reply:
Yes, just mention the plt.axes() syntax and the users can choose whatever they prefer